### PR TITLE
[SUBMARINE-199] Add instructions for working with CICD via python script

### PR DESCRIPTION
**Pull request by liuxunorg**
### What is this PR for?
Currently submarine supports CICD through docker.
In the absence of a docker environment, merge PR operations can also be performed by means of python scripts.
Adding this documentation will also help users understand the submarine merge PR execution process.

### What type of PR is it?
[Documentation]

### What is the Jira issue?
* https://issues.apache.org/jira/browse/SUBMARINE-199

### How should this be tested?
* [CI Pass](https://travis-ci.org/liuxunorg/hadoop-submarine/builds/589737584)

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes

### CODE REVIEW

File: dev-support/cicd/README.md [hunk 1]
```diff
@@ -34,3 +41,18 @@ Enter Menu ID:1
 ==== Merge PR Begin ====
 Enter Your Apache JIRA User name:
 ```
+
+## Manual mode
+
+### The Procedure of Merging PR
+
+1. mkdir ${work_dir} & cd ${work_dir}
```
Comment (tangzhankun) [inline dev-support/cicd/README.md]: & -> &&?

File: dev-support/cicd/README.md [hunk 2]
```diff
@@ -34,3 +41,18 @@ Enter Menu ID:1
...
+2. git clone https://git-wip-us.apache.org/repos/asf/hadoop-submarine
```
Comment (tangzhankun) [inline dev-support/cicd/README.md]: git clone https://gitbox.apache.org/repos/asf/hadoop-submarine.git

File: dev-support/cicd/README.md [hunk 3]
```diff
@@ -34,3 +41,18 @@ Enter Menu ID:1
 ==== Merge PR Begin ====
 Enter Your Apache JIRA User name:
 ```
+
+## Manual mode
+
+### The Procedure of Merging PR
+
```
Comment (tangzhankun) [inline dev-support/cicd/README.md]: add "install python 2.7.13 and pip insalll jira"?

Comment (liuxunorg) [reply dev-support/cicd/README.md]: Done.

File: dev-support/cicd/README.md (hunk 1 above)
Comment (liuxunorg) [reply dev-support/cicd/README.md]: Done.

File: dev-support/cicd/README.md (hunk 2 above)
Comment (liuxunorg) [reply dev-support/cicd/README.md]: Done.

File: dev-support/cicd/README.md [hunk 4]
```diff
@@ -34,3 +41,22 @@ Enter Menu ID:1
 ==== Merge PR Begin ====
 Enter Your Apache JIRA User name:
 ```
+
+## Manual mode
+
+First, You need install `python 2.7.13` and `pip insalll jira`
```
Comment (tangzhankun) [inline dev-support/cicd/README.md]: There's a  typo. The "installl" should be "install"

**Comment by jiwq** [comment]
+1 for the `959e231` commit

File: dev-support/cicd/README.md (hunk 4 above)
Comment (tangzhankun) [reply dev-support/cicd/README.md]: @liuxunorg , could you please update the typo ?

Comment (liuxunorg) [reply dev-support/cicd/README.md]: Done.

**Comment by liuxunorg** [comment]
@tangzhankun Please help me merge PR, Thank you.

**Comment by tangzhankun** [comment]
@liuxunorg , committed. Thanks for the patch. And thanks for the review. @jiwq

**Pull request closed without merging**